src/SerialFirmata.h

```diff
@@ -23,6 +23,10 @@
 #include <SoftwareSerial.h>
 #endif
 
+//uncomment SERIAL_STORE_AND_FORWARD to collect bytes received by serial port until the receive buffer 
+// gets filled or a data gap is detected to avoid forwarding single bytes at baud rates below 50000
+//#define SERIAL_STORE_AND_FORWARD
```

Comment:
Just had another idea: Firmata uses the term "serial pin" for its non-host serial ports. So maybe we should call it ```FIRMATA_SERIAL_RX_PIN_BUFFERING```?